src/app/screens/Transactions/index.tsx

```diff
@@ -15,6 +16,7 @@ function Transactions() {
 
   const { settings, getFormattedFiat } = useSettings();
   const [transactions, setTransactions] = useState<Transaction[]>([]);
+  const [transactionsLoading, setTransactionsLoading] = useState<boolean>(true);
```

Comment:
I think the default state should be `false`and `setTransactionsLoading(true)` should be called in `fetchData`. That way the loading state would also appear with repeated loading.

Reply:
I updated accordingly.